signature/fixtures/pki_roots_crt.pem

```diff
@@ -0,0 +1,33 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
These are all freshly-generated, aren’t they?

Seeing `C=ES, L=Valencia, O=IT, OU=Security, CN=Linuxera Root Certificate Authority` in there was a bit surprising.

\*shrug\* the names don’t really matter, probably not worth re-doing all of that.

Reply:
I haven't addressed this now, but can do this in a follow-up PR.